internal/recover/recover.go

```diff
@@ -0,0 +1,339 @@
+package recover
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path"
+	"strings"
+
+	dqlite "github.com/canonical/go-dqlite/client"
+	"github.com/canonical/lxd/shared/logger"
+	"gopkg.in/yaml.v2"
+
+	"github.com/canonical/microcluster/cluster"
+	"github.com/canonical/microcluster/internal/sys"
+	"github.com/canonical/microcluster/internal/trust"
+)
+
+// GetLocalClusterMembers parses the trust store and
+// path.Join(filesystem.DatabaseDir, "cluster.yaml").
+func GetLocalClusterMembers(filesystem *sys.OS) ([]cluster.LocalMember, error) {
```

Comment:
Can this be renamed to `GetDqliteClusterMembers` instead? `LocalClusterMembers` might clash with [types.ClusterMemberLocal](https://github.com/canonical/microcluster/blob/f79afecf8b8945cd4500d3d0546e52671cc79aac/internal/rest/types/cluster.go#L23)

Reply:
Yeah, let's do `s/GetLocalClusterMembers/GetDqliteClusterMembers` and `s/LocalMember/DqliteMember/`. Think I will stick with `--local` for the cli though unless you have objections; I think it makes more sense than using `--dqlite` in that context.